integration/app/test-demo/README.md

```diff
@@ -0,0 +1,76 @@
+# README - React Testing Demo
+
+The React Testing Demo is used to test the Amazon Chime SDK React Components Library.
+
+Different tests may require different testing demos. Creating a new testing demo for each test would add extra maintenance work. We don't like it.
+
+Following the example of [WebRTC samples](https://webrtc.github.io/samples/), we come up with the concept of a Mini Test App - a minimalist, single-page application containing only the necessary components for a single test. All Mini Test Apps are sub-pages of the React Testing Demo.
+
+## Guideline
+
+When adding tests for a group of related features, build a new Mini Test App.
+
+* For example, we already have a "Roster Test App" and you want to add tests for video features, you should create a "Video Test App".
+
+If there is already an existing Mini Test App for the feature that you are working on, you may add onto the Mini Test App.
+
+* For example, since we already have a "Roster Test App" - if you want to add tests for a `useRosterSize` hook which returns the size of the roster, you should add the test to the "Roster Test App".
+
+## Run the React Testing Demo Locally
+
+1. If you haven't already, [configure your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+2. Install the dependencies: `npm install`.
+3. Start the webpack server and node server concurrently: `npm start`.
+4. Open <http://localhost:9000/> in your browser.
+
+## Add a New Mini Test App
+
+1. Implement a new Mini Test App under `pages` folder.
+   * You could reuse the components and containers to facilitate your development.
+
+2. Create a new route for Mini Test App in `src/app.tsx`.
+
+    ```js
+    const App: React.FC = () => (
+      <ThemeProvider theme={lightTheme}>
+        <Router>
+          <Routes>
+            <Route path={routes.HOME} element={<Home />} />
+            <Route path={routes.ROSTER_TEST} element={<RosterTestApp />} />
+            {/* Add <Route> to new Mini Test App below */}
+          </Routes>
+        </Router >
+      </ThemeProvider>
+    );
+
+    export const routes = {
+      HOME: '/',
+      ROSTER_TEST: 'roster-test',
+      // Add new routes below
+    };
+    ```
+
+3. Add the new Mini Test App to `pages/Home.tsx`.
+   * Add the test coverage of Mini Test App following the example.
+
+    ```tsx
+    export const Home: React.FC = () => (
+      <>
+        <Heading level={4}>React Component Library Test Demo</Heading>
+        <TestApps>
+          <TestApp
+            name="Roster Test"
```

Comment:
For strings, we usually use ' instead of " (at least in React library).

Reply:
Forgot update the code here. Nice catch. Updated.